week1/week1/src/main/java/com/livenow/week1/controller/dto/MemberAddRequestDto.java

```diff
@@ -0,0 +1,17 @@
+package com.livenow.week1.controller.dto;
+
+import com.livenow.week1.domain.Member;
+import com.livenow.week1.domain.Team;
+import lombok.Getter;
+
+@Getter
+public class MemberAddRequestDto {
+
+    private Long teamId;
+    private Long memberId;
+
+    public MemberAddRequestDto(Team teamEntity, Member memberEntity) {
```

Comment:
`Xxx`객체의 인스턴스는 `xxxEntity`보다는 `xxx`가 낫지 않을까?
```suggestion
    public MemberAddRequestDto(Team team, Member member) {
```

Reply:
수정 완료